### PR TITLE
RavenDB-16479 Add time series to database stats page

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/status/statistics.ts [hunk 1]
```diff
@@ -62,13 +62,32 @@ class statistics extends viewModelBase {
 
         popoverUtils.longWithHover($(".js-cv-tooltip"),
             {
-                content: `<div>${cvTooltip}</div>`
+                content: this.stats().databaseChangeVector.length == 0 ? "" : `<div>${cvTooltip}</div>`
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/status/statistics.ts]: use `===`

File: src/Raven.Studio/typescript/models/database/stats/statistics.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ class statistics {
     countOfCompareExchange: string;
     countOfRevisions: string;
     countOfCounters: string;
+    countOfTimeSeriesSegments:string;
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/stats/statistics.ts]: space after `:`

Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/stats/statistics.ts]: done

File: src/Raven.Studio/typescript/viewmodels/database/status/statistics.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/status/statistics.ts]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**